conda/base/context.py

```diff
@@ -710,27 +748,90 @@ def root_writable(self):
         return False
 
     @property
-    def envs_dirs(self):
-        return mockable_context_envs_dirs(
+    def envs_dirs(self) -> tuple[IndexedSet]:
+        """Get the env_dirs for the environment.
+
+        :return: Directories where envs are stored
+        """
+        if self._envs_dirs:
+            # User has already specified what directories to use. Use the legacy behavior
+            # (which in this case _doesn't_ mean just use the directories specified in
+            # the configuration)
+            return self._prefix_envs_dirs()
+
+        prefix_dir = self._root_prefix_envs()
+        if isdir(prefix_dir) and len(os.listdir(prefix_dir)) > 0:
+            # Prefix location is in use; emit warning message.
+            log.warning(
+                "conda is using the prefix location for the envs directory "
```

Comment:
embedded location? Prefix in conda is overloaded.

Reply:
Hmm, how about "root prefix"? That's most accurate, I think.